doc/rados/configuration/common.rst

```diff
@@ -186,33 +193,30 @@ Example ceph.conf
 Naming Clusters (deprecated)
 ============================
 
-Each Ceph cluster has an internal name that is used as part of configuration
-and log file names as well as directory and mountpoint names.  This name
-defaults to "ceph".  Previous releases of Ceph allowed one to specify a custom
-name instead, for example "ceph2".  This was intended to facilitate running
-multiple logical clusters on the same physical hardware, but in practice this
-was rarely exploited and should no longer be attempted.  Prior documentation
-could also be misinterpreted as requiring unique cluster names in order to
-use ``rbd-mirror``.
+Each Ceph cluster has an internal name. This internal name is used as part of
+configuration, and as part of "log file" names as well as part of directory
+names and as part of mountpoint names. This name defaults to "ceph". Previous
+releases of Ceph allowed one to specify a custom name instead, for example
+"ceph2". This option was intended to facilitate the running of multiple logical
+clusters on the same physical hardware, but in practice it was rarely
+exploited. Custom cluster names should no longer be attempted. Old
+documentation might lead readers to wrongly think that unique cluster names are
+required to use ``rbd-mirror``. They are not required.
 
 Custom cluster names are now considered deprecated and the ability to deploy
-them has already been removed from some tools, though existing custom name
-deployments continue to operate.  The ability to run and manage clusters with
-custom names may be progressively removed by future Ceph releases, so it is
-strongly recommended to deploy all new clusters with the default name "ceph".
+them has already been removed from some tools, although existing custom-name
+deployments continue to operate. The ability to run and manage clusters with
+custom names might be progressively removed by future Ceph releases, so **it is
```

Comment:
s/might/will/

If it hasn't already

Reply:
unfinished business

Reply:
@zdover23 (of the future): Which was the last version of Ceph to support custom cluster names?